plugins/discovery-gce/build.gradle

```diff
@@ -7,19 +7,30 @@ esplugin {
 }
 
 versions << [
-  'google': '1.23.0'
```

Comment:
changes copied and pasted from main : https://github.com/elastic/elasticsearch/blob/main/plugins/discovery-gce/build.gradle